template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/assets/ml-artifacts-asset.yml.tmpl

```diff
@@ -31,24 +31,24 @@ targets:
           comment: Registered model in Unity Catalog for the "{{template `project_name` .}}" ML Project for ${bundle.target} deployment target.{{end}}
 
   prod:
-    resources:
+    assets:
```

Comment:
We can't rename these since these are special keywords reserved by DABs, revert please.